gammagl/datasets/acm4heco.py

```diff
@@ -0,0 +1,258 @@
+
+from gammagl.datasets import HGBDataset, IMDB
+import scipy.sparse as sp
+import json
+import ssl
+import urllib.request
+import sys
+import os
+import os.path as osp
+from collections import Counter
+from typing import Callable, List, Optional
+import numpy as np
+import tensorlayerx as tlx
+import shutil
+import errno
+from gammagl.data import (HeteroGraph, InMemoryDataset, extract_zip)
+from sklearn.preprocessing import OneHotEncoder
+import requests
+import io
+import zipfile
+from gammagl.data import download_url
+r"""The heterogeneous ACM dataset from the `"Self-supervised Heterogeneous Graph Neural Network with

```

Comment:
Error location! Refer to the "document generation guide" and test on your local laptop. This docstring will not generate correctly.